docs/section23/Rule23-5.md

```diff
@@ -0,0 +1,33 @@
+
+# Airside System - Rule 23-5 
+
+**Schema Version:** 0.0.34  
+**Mandatory Rule:** True  
+**Rule ID:** 23-5  
+**Rule Description:** For baseline systems 6 and 8, Fans in parallel VAV fan-powered boxes shall run as the first stage of heating before the reheat coil is energized.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**90.1 Section Reference:** G3.1.3.14 Fan Power and Control (Systems 6 and 8)  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-6 or 8.  
+
+**Function Calls:**  
+
+1. get_baseline_system_types()
+2. baseline_system_type_compare()
+
+**Applicability Checks:**  
+- create a list of the target system types: `APPLICABLE_SYS_TYPES = [HVAC_SYS.SYS_6, HVAC_SYS.SYS_8]`
```

Comment:
I would consider making this a primary rule to avoid having "undetermined" outcome for all projects with HVAC systems 6 & 8. We seems to be missing a schema element to accomplish it. Perhaps add Terminal.fan_powered_box_heat_first_stage. I'll added it as an issue to the schema.

Reply:
update the variable name to: `is_first_stage_heat_fan_powered_box`